docs/pages/setup/reference/license.mdx

```diff
@@ -21,9 +25,16 @@ The `license_file` path can be either absolute or relative to the configured
 `data_dir`. If the license file path is not set, Teleport will look for the
 `license.pem` file in the configured `data_dir`.
 
+The Teleport l
+
 <Admonition
   type="tip"
   title="NOTE"
 >
-  Only Auth Servers require the license. Proxies and Nodes that do not also have Auth role enabled do not need the license.
+  Only Auth Servers require the license. Proxies and Nodes that do not also have Auth role enabled do not need the license.  
```

Comment:
Rather than expand this Admonition box, I think we can remove the `Admonition` tags and make it regular text instead. Since it's a short page, I don't think it will be very easy to skip a paragraph.